Core/dune

```diff
@@ -1,16 +1,12 @@
 (library
  (name core_plugin)
  (public_name quickchick.core_plugin)
- (modules :standard)
- (libraries show_lib coq.plugins.ltac coq.plugins.extraction)
-)
+ (flags :standard -rectypes)
+ (libraries coq.plugins.extraction))
 
-(rule
- (targets g_quickchick.ml)
- (deps (:pp-file g_quickchick.mlg))
- (action (run coqpp %{pp-file})))
+(coq.pp (modules g_quickchick))
 
-(coqlib
+(coq.theory
```

Comment:
`coq.theory` and `package` are additions in Dune 1.10, you can go back to `coqlib` and `public_name` if that's a hassle for you.